monkestation/code/game/objects/effects/spawners/roomspawner.dm

```diff
@@ -12,7 +12,9 @@
 	dedicated_rooms = list(/datum/map_template/random_room/box_default_bar = 1,
 							/datum/map_template/random_room/box_clown_bar = 1,
 							/datum/map_template/random_room/box_syndi_bar = 1,
-							/datum/map_template/random_room/box_vietmoth_bar = 1)
+							/datum/map_template/random_room/box_vietmoth_bar = 1,
+							/datum/map_template/random_room/box_bloody_bar = 1,
```

Comment:
Actual cult stuff would be either dangerous to the crew, or an easy source of free runed metal.
Tables and chairs should be wooden, since that's what Nar'sie transforms them into.

Reply:
cult walls are the artificer versions and dont drop any girders or metal.